File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/GameLevelListResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class GameLevelListResponse 
{
    public int Start { get; set; }
    public int Limit { get; set; }
    public long CountTotal { get; set; }
    public List<GameLevelResponse> Items { get; set; }
    public int Count { get; set; }
    public bool HasMoreResults { get; set; }
}

File: server-scala-play-server/app/model/GameLevelListResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for GameLevelListResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-11T19:31:43.310890579Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class GameLevelListResponse(
  start: Option[Int],
  limit: Option[Int],
  countTotal: Option[Long],
  items: Option[List[GameLevelResponse]],
  count: Option[Int],
  hasMoreResults: Option[Boolean]
)

object GameLevelListResponse {
  implicit lazy val gameLevelListResponseJsonFormat: Format[GameLevelListResponse] = Json.format[GameLevelListResponse]
}

